node/tessel-export.js

```diff
@@ -408,17 +410,22 @@ Tessel.Port.PATH = {
   'B': '/var/run/tessel/port_b'
 };
 
-Tessel.Pin = function(pin, port, interruptSupported, analogSupported) {
+Tessel.Pin = function(pin, port, interruptSupported, analogSupported, pullSupported) {
   this.pin = pin;
   this._port = port;
   this.interruptSupported = interruptSupported || false;
   this.analogSupported = analogSupported || false;
+  this.pullSupported = pullSupported || false;
   this.interruptMode = null;
   this.isPWM = false;
 };
 
 util.inherits(Tessel.Pin, EventEmitter);
 
+Tessel.Pin.adcCapablePins = [4, 7];
+Tessel.Pin.pullCapablePins = [2, 3, 4, 5, 6, 7];
+Tessel.Pin.interruptCapablePins = [2, 5, 6, 7];
```

Comment:
Eventually I'd like to revisit this (the entire pin capability organization) for refactoring, but for now it's fine. I still need to do a smoke test, which I'll try to do today